nixos/tests/userborn.nix

```diff
@@ -22,6 +22,7 @@ in
 
   nodes.machine = {
     services.userborn.enable = true;
+    boot.initrd.systemd.enable = true;
```

Comment:
This was the only test that didn't had initrd.systemd enabled. However as in my motivation it's unclear how other services that currently rely on activation scripts can depend on the service being run. It would introduce some additional complexity to support both systemd stage1 + userborn vs. only userborn.